lib/puppet/type/dsc_xcomputer.rb

```diff
@@ -48,9 +48,23 @@
   newparam(:name, :namevar => true ) do
   end
 
+  #newproperty(:last_dsc_result) do
+  #end
```

Comment:
A plain old Ruby property `attr_reader :last_dsc_result` should be fine here.  I don't think we ever intend to expose these raw values to Puppet in any way.